website/components/PluginSupportStatusTable.tsx

```diff
@@ -204,7 +204,7 @@ const pluginSupportStatusList: PluginSupportStatus[] = [
   },
   {
     name: 'RequireJsStuffPlugin',
-    status: SupportStatus.NotSupported,
```

Comment:
Shouldn't change this, we don't have this export, so users can't use this by plugin